spec/kitchen/provisioner/salt_solo_spec.rb

```diff
@@ -113,8 +113,20 @@
   describe '#init_command' do
     subject { provisioner.init_command }
 
-    it 'should give a sane command' do
-      is_expected.to match(/mkdir/)
+    context 'when unix' do
+      it 'should give a sane command' do
+        is_expected.to match(/mkdir/)
+      end
+    end
+
+    context 'when windows' do
+      let(:platform) do
+        instance_double(Kitchen::Platform, os_type: 'windows')
+      end
+   
+      it 'should use powershell' do
+        is_expected.to match(/mkdir -Path/)
```

Comment:
@lomeroe we'll need to update this if #105 gets merged to expect `mkdir -Force -Path`